include/Skill/Progression.h

```diff
@@ -49,8 +49,8 @@ namespace CC
 		static const int neededXpFunction(int level);
 		int neededXpToLevelUp() const;
 		bool addXp(int xp);
-		void addSkillPoints(int skillPoints);
-		void addPointToSkill(Skill skill, int amountOfSkillpoints);
+		void addSkillPoints(int amount);
+		void addPointToSkill(Skill skill, int amount);
 		void enemyKilled(int xp);
```

Comment:
If you want to make a little of documentation quickly write `/**` and then enter. Clion will create Doxygen type of comment, very useful when you want to describe a function

Reply:
ok...

Reply:
It's just so you know